test/cmake/seqan3_require_test.cmake

```diff
@@ -5,93 +5,28 @@
 # shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
 # -----------------------------------------------------------------------------------------------------
 
-cmake_minimum_required (VERSION 3.10)
+cmake_minimum_required (VERSION 3.16)
```

Comment:
why not 3.16.`9`?

Reply:
We could do 3.16.3 because that's what Google requires, but we usually don't do patch level. No strong feelings